src/store/model/amazon-de.ts

```diff
@@ -23,34 +24,34 @@ export const AmazonDe: Store = {
 			url: 'https://www.amazon.de/dp/B083JX52VG/'
 		},
 		{
-			brand: 'pny',
-			model: 'xlr8',
+			brand: 'asus',
```

Comment:
I find it interesting that you moved a lot of these in lexicographical order, but then some not. Am I reading this right?

Reply:
yes i started to normalize and sort them somewhere in the middle of my contributions and forgot about them, sorry.

Reply:
i actually don't know what i tried here first, no shop except one was in correct order.